testSuite/standardTF_reflectivityCalculationTest/testReflectivityCalculations.m

```diff
@@ -0,0 +1,187 @@
+classdef testReflectivityCalculations < matlab.unittest.TestCase
+%%
+% testReflectivityCalculations Class based unit tests for RAT API, the
+% reflectivity calculation and pre- and post-processing routines.
+%
+% In this class, we test: RAT, RATMain, singleCalculation,
+% reflectivityCalculationWrapper, reflectivityCalculation,
+% RatParseClasstoStructs_new, parseResultToStruct,
+% RATParseOutToProjectClass
+%
+% We are using the test cases for a standard TF reflectivity calculation
+% to test the routines. We consider standard layers, custom layers and
+% custom XY examples. For the reflectivity calculation itself, we consider
+% the serial and parallel versions (both points and contrasts), using both
+% the MATLAB and compiled (MEX) versions.
+%
+% Paul Sharp 19/01/23
+%
+%% Declare properties and parameters
```

Comment:
To the best of my knowledge, the section break comments (%%) don't work in test classes, so they can be removed, just put comment in function as usual

Reply:
I'd put the breaks in for readability of the source code as much as anything, so I'll leave them for now.